tests/backup/backup_multiple_provisioner_test.go

```diff
@@ -222,3 +222,407 @@ var _ = Describe("{MultipleProvisionerCsiSnapshotDeleteBackupAndRestore}", func(
+		scheduleUid                               string
+		srcClusterUid                             string
+		schedulePolicyInterval                    = int64(15)
+		scheduledAppContextsForMultipleAppSinleNs []*scheduler.Context
+		scheduledAppContextsForDefaultVscBackup   []*scheduler.Context
+		scheduledAppContextsForCustomVscBackup    []*scheduler.Context
+		allAppContext                             []*scheduler.Context
+		appSpecListDefaultVsc                     []string
+		appSpecListCustomVsc                      []string
+		defaultSchBackupName                      string
+		scheduleList                              []string
+		defaultProvisionerScheduleName            string
+		nonDefaultVscSchBackupName                string
+		randomStringLength                        = 10
+		forceKdmpSchBackupName                    string
+		kdmpScheduleName                          string
+		multipleProvisionerSameNsScheduleName     string
+		multipleNsSchBackupName                   string
+		preRuleNameMultiProvisioner               string
+		postRuleNameMultiProvisioner              string
+		preRuleUidMultiProvisioner                string
+		postRuleUidMultiProvisioner               string
+		appSpecListMultiProvisioner               []string
+		preRuleNameDefaultVsc                     string
+		postRuleNameDefaultVsc                    string
+		preRuleUidDefaultVsc                      string
+		postRuleUidDefaultVsc                     string
+		preRuleNameCustomVsc                      string
+		postRuleNameCustomVsc                     string
+		preRuleUidCustomVsc                       string
+		postRuleUidCustomVsc                      string
+		clusterProviderName                       = GetClusterProvider()
+		provisionerDefaultSnapshotClassMap        = GetProvisionerDefaultSnapshotMap(clusterProviderName)
+		provisionerSnapshotClassMap               = GetProvisionerSnapshotClassesMap(clusterProviderName)
+	)
+
+	JustBeforeEach(func() {
+		if GetClusterProvider() != "ibm" || GetClusterProvider() != "openshift" {
+			// This test is meant to run only on IBM later will enable on other configs
+			log.Infof("Skipping the test.This test is currently configured to run on IBM and openshift and environments. Future iterations will enable on other configs")
+			Skip("Skipping the test.This test is currently configured to run on IBM and openshift environments. Future iterations will enable on other configs")
+		}
+		StartPxBackupTorpedoTest("MultipleProvisionerCsiKdmpBackupAndRestore", "Backup and restore of namespaces with multiple provisioners using csi with volume class mapping and force kdmp", nil, 296724, Sn, Q4FY24)
+		backupLocationMap = make(map[string]string)
+		labelSelectors = make(map[string]string)
+		providers = GetBackupProviders()
+		// Deploy multiple application in a single namespace using different provisioner
+		taskName := fmt.Sprintf("%s-%s", TaskNamePrefix, RandomString(randomStringLength))
```

Comment:
Can we use time as well to be sure that the string doesn't fail if ran again?

Reply:
done